prepack.js

```diff
@@ -1,34 +1,11 @@
 #!/usr/bin/env node
 'use strict';
 
-// add this check because withFileTypes was added in 10.10.0. for simplicity
-// just require node 12.
-if (process.versions.node.split('.').shift() < 12) {
-  // eslint-disable-next-line no-console
-  console.error(`appoptics-apm must be released using node >= 12; using ${process.version}`);
-  process.exit(1);
-}
-
 const fs = require('fs');
-
 const files = fs.readdirSync('.', 'utf8');
 
 let errorCount = 0;
 
-//
-// make sure that there are no ubuntu-* files left around
```

Comment:
where did these come from?
I guess it was some docker artifact and you're not creating them anymore

Reply:
`ubuntu-*` files were (are) the output from running `testeachversion`. We no longer need to test this via script because a) we use an explicit file list and b) we will be running this in GH Actions moving forward.

There may have been also other "generators" of such files but they are no longer present in our setup.